modules/brainbrowser/php/MincEnv.php.inc

```diff
@@ -27,8 +27,18 @@
 
 require_once "NDB_Config.class.inc";
 
-$config    =& NDB_Config::singleton();
-$minc_path = $config->getSetting("MINCToolsPath");
+$config =& NDB_Config::singleton();
+try {
+    $minc_path = $config->getSetting("MINCToolsPath") ?? '';
+} catch (ConfigurationException $e){
```

Comment:
This try/catch shouldn't be here. getSetting throws an exception by design. A setting that's being used not being configured is an exceptional circumstance indicating the server is misconfigured, and shouldn't be handled unless there's a good reason that the config might not exist in normal circumstances.